js/bybit.js

```diff
@@ -6283,7 +6283,13 @@ module.exports = class bybit extends Exchange {
         const size = Precise.stringAbs (this.safeString (position, 'size'));
         let side = this.safeString (position, 'side');
         if (side !== undefined) {
-            side = (side === 'Buy') ? 'long' : 'short';
+            if (side === 'Buy') {
```

Comment:
@sc0Vu don't know if you noticed but it was already protected against the default "short" check this PR:  https://github.com/ccxt/ccxt/pull/16424.

I added this `if (side !== undefined) {}` so here `side = (side === 'Buy') ? 'long' : 'short';` we know side is not undefined

Reply:
@carlosmiei I saw this response from user ; `{'info': {'positionIdx': '0', 'riskId': '1', 'symbol': 'BTCUSDT', 'side': 'None', 'size': '0.000', 'positionValue': '0', 'entryPrice': '0', 'tradeMode': '1', 'autoAddMargin': '0', 'leverage': '3', 'positionBalance': '0', 'liqPrice': '0.00', 'bustPrice': '0.00', 'takeProfit': '0.00', 'stopLoss': '0.00', 'trailingStop': '0.00', 'unrealisedPnl': '0', 'createdTime': '1672959618477', 'updatedTime': '1673273970695', 'tpSlMode': 'Full', 'riskLimitValue': '2000000', 'activePrice': '0.00', 'markPrice': '17225.05', 'cumRealisedPnl': '0', 'positionMM': '', 'positionIM': '', 'positionStatus': 'Normal', 'sessionAvgPrice': '0.00', 'occClosingFee': '0', 'avgPrice': '0'}, 'id': None, 'symbol': 'BTC/USDT:USDT', 'timestamp': 1673365144571, 'datetime': '2023-01-10T15:39:04.571Z', 'initialMargin': None, 'initialMarginPercentage': None, 'maintenanceMargin': None, 'maintenanceMarginPercentage': None, 'entryPrice': None, 'notional': 0.0, 'leverage': 3.0, 'unrealizedPnl': None, 'contracts': 0.0, 'contractSize': 1.0, 'marginRatio': None, 'liquidationPrice': None, 'markPrice': 17225.05, 'collateral': 0.0, 'marginMode': 'isolated', 'side': 'short', 'percentage': None}`, wonder did server return string `'None'?

If it's string `'None'`, the side probably could be `short`.

Reply:
@sc0Vu @Osthumus yeah u're both right, I didn't notice the string could be literally 'None'. Going to merge this asap